src/java/org/apache/cassandra/schema/MigrationManager.java

```diff
@@ -213,6 +213,11 @@ public static void announce(Collection<Mutation> schema)
         FBUtilities.waitOnFuture(f);
     }
 
+    public static Future<?> announceWithoutPush(Collection<Mutation> schema)
+    {
+        return MIGRATION.submit(() -> Schema.instance.mergeAndAnnounceVersion(schema));
+    }
+
```

Comment:
This commit would be pushed to CASSANDRA-16387 eventually, but it's easier to test here for now.